src/cascadia/TerminalConnection/ConhostConnection.h

```diff
@@ -24,20 +24,21 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         winrt::event<TerminalConnection::TerminalOutputEventArgs> _outputHandlers;
         winrt::event<TerminalConnection::TerminalDisconnectedEventArgs> _disconnectHandlers;
 
-        uint32_t _initialRows;
-        uint32_t _initialCols;
-        hstring _commandline;
-        hstring _startingDirectory;
+        uint32_t _initialRows{};
+        uint32_t _initialCols{};
+        hstring _commandline{};
+        hstring _startingDirectory{};
 
-        bool _connected;
-        HANDLE _inPipe;  // The pipe for writing input to
-        HANDLE _outPipe; // The pipe for reading output from
-        HANDLE _signalPipe;
+        bool _connected{};
+        HANDLE _inPipe{ INVALID_HANDLE_VALUE };  // The pipe for writing input to
+        HANDLE _outPipe{ INVALID_HANDLE_VALUE }; // The pipe for reading output from
+        HANDLE _signalPipe{ INVALID_HANDLE_VALUE };
         //HPCON _hPC;

```

Comment:
is `_hPC` dead code?

Reply:
I guess? It was already there before I changed this.

Reply:
alas, it is. feel free to remove!